src/config.rs

```diff
@@ -167,6 +170,28 @@ pub trait Config: Sized + Send + Sync + UnwindSafe + RefUnwindSafe + 'static {
         vec![Biome::default()]
     }
 
+    fn commands(&self) -> Commands {
```

Comment:
What happens if you send multiple `Commands` packets? Do subsequent packets delete the set of commands and replace them with new ones? If that's the case then adding commands should be done with `Client` and not `Config`.

Is it possible to construct a bad `Commands` packet using this that would disconnect players? If it is then we need to be doing validation to ensure that doesn't happen.